src/main/java/com/gildedrose/Item.java

```diff
@@ -14,6 +14,18 @@ public Item (String name, int sellIn, int quality) {
         this.quality = quality;
     }
 
+    public void updateQuality() {
```

Comment:
Something else to consider: this method is called `updateQuality` but that's misleading because it also updates the `sellIn` value. What could you do to improve this?